executor/executor.go

```diff
@@ -257,17 +257,26 @@ func (e *ShowDDLExec) Next(ctx context.Context, chk *chunk.Chunk) error {
 	}
 
 	ddlJobs := ""
+	originalQuery := ""
```

Comment:
how about s/originalQuery/query/ ?

Reply:
I think originalQuery is clear for developer , and QUERY is clear for user.
Therefore , I use QUERY for user , and originalQuery for developer.
If you think unified them is good , I will do that.